pages/learn/manage/billing.md

```diff
@@ -52,6 +52,8 @@ For both devices and users, usage is calculated based on the maximum number in y
 
 As an example, let's say you are on the Pilot plan, which includes up to 50 devices and 3 users. In one month, your active device count steadily climbs from 43 to 52. During this same month, you temporarily increase your user count from 3 to 4, before returning it to 3 at the end of the month. Based on this usage, your dynamic billing charges for the month will include an additional 2 devices and 1 user.
 
+Furthermore we charge a device or user only once for the most feature rich usage tier within the same billing period. For example, when a user had Developer role in the first half of the billing period and then only Operator role in the second half, we would charge this usage as Developer role for the billing period. This is because the Developer role has more features than the Operator role as seen in [application membership types][app-membership-types]. It's important to note that we are not double charging for the same user.
```

Comment:
```suggestion
Furthermore, we charge a device or user only once for the most feature-rich usage tier within the same billing period. For example, if a user was assigned a Developer role for the first half of the billing period and was later reassigned the Operator role for the second half of the billing period. In that scenario, we would charge the usage as a Developer role for the entire billing period. This is because the Developer role is a more feature-rich usage tier than the Operator role, as seen in [application membership types][app-membership-types]. It's important to note that we are not double charging for the same user.
```

Reply:
Thanks for the suggestions @vipulgupta2048. Definitely reads better. I committed a slightly modified version. Please take another look.